packages/shared-ux/page/analytics_no_data/src/__snapshots__/analytics_no_data_page.component.test.tsx.snap

```diff
@@ -145,8 +145,127 @@ exports[`AnalyticsNoDataPageComponent renders correctly 1`] = `
           >
             <EuiLoadingSpinner>
               <span
-                className="euiLoadingSpinner euiLoadingSpinner--medium"
-              />
+                aria-label="Loading"
```

Comment:
The bloat here will need to be resolved on our end, either with a `shallow` render or a mock. cc: @majagrubic @sebelga